### PR TITLE
Add writeMessage to mirror readMessage from CodedBufferReader.

### PR DESCRIPTION
**Pull request by Nash0x7E2**
Add writeMessage to mirror readMessage from CodedBufferReader.

### CODE REVIEW

File: protobuf/lib/src/protobuf/coded_buffer_writer.dart [hunk 1]
```diff
@@ -383,13 +383,17 @@ class CodedBufferWriter {
...
       case PbFieldType._MESSAGE_BIT:
-        final mark = _startLengthDelimited();
-        value.writeToCodedBufferWriter(this);
-        _endLengthDelimited(mark);
+        writeMessage(value);
         break;
     }
   }
 
+  void writeMessage(GeneratedMessage message) {
```
Comment (sigurdm) [inline protobuf/lib/src/protobuf/coded_buffer_writer.dart]: This needs documentation to make clear that the size is being written. According to: https://developers.google.com/protocol-buffers/docs/techniques#streaming an encoded message does not encode its own size.

I think this should be called writeMessageNoTag to correspond to https://github.com/protocolbuffers/protobuf/blob/master/java/core/src/main/java/com/google/protobuf/CodedOutputStream.java#L1238 that does more or less the same thing.

**Comment by szakarias** [comment]
@Nash0x7E2 , are you planning on moving forward with this PR?

**Comment by Nash0x7E2** [comment]
@szakarias I am unable to work on this PR at the moment, can it be closed?

**Comment by szakarias** [comment]
Yes I can close it. Feel free to make a new PR in the future . :-)

**Pull request closed without merging**